packages/libretro/snes9x2005_plus/patches/snes9x2005_plus-01-enable_blargg_apu.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index f5891c4..54a5368 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,7 +1,7 @@
+ DEBUG                 = 0
+ PERF_TEST             = 0
+ LOAD_FROM_MEMORY_TEST = 1
+-USE_BLARGG_APU        = 0
++USE_BLARGG_APU        = 1
```

Comment:
Can this change be made just by passing USE_BLARGG_APU=1 to the `make` command?

I think you do that with `PKG_MAKE_OPTS_TARGET` in [`package.mk`](https://wiki.libreelec.tv/package.mk).

```
PKG_MAKE_OPTS_TARGET="USE_BLARGG_APU=1"
```